src/components/network/network-panel.js

```diff
@@ -45,20 +45,30 @@ genotet.NetworkPanel.prototype.initPanel = function() {
         }.bind(this));
   }, this);
 
+  // Input type update
+  this.container.find('#regex input').click(function() {
+    this.isRegex_ = true;
+  }.bind(this));
+  this.container.find('#string input').click(function() {
+    this.isRegex_ = false;
+  }.bind(this));
+  this.container.find('#regex input').trigger('click');
```

Comment:
Sorry about that. I am not sure about this, I just copied the code for expression manipulations. Could you give me more instructions about this?